cudamapper/CMakeLists.txt

```diff
@@ -67,7 +67,15 @@ target_include_directories(${MODULE_NAME}
         $<INSTALL_INTERFACE:include>
         $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
 )
-target_link_libraries(${MODULE_NAME} gwbase gwio cub)
+
+# optionally link against htslib
+if (gw_build_htslib)
+    add_compile_definitions(GW_BUILD_HTSLIB)
```

Comment:
I think this is the newer way to handling compile definitions - https://cmake.org/cmake/help/latest/command/target_compile_definitions.html

Reply:
So this definition is used in multiple targets (libcudamapper, sample_cudamapper, and cudamapper-bin), so I think defining this for the module gives us the benefit of not having to add this define for each target. That said, I defer to you on this (it's just a few more lines of cmake here and there)

Reply:
okay I see, in that case this is fine. Thanks!